modules/client/examples/index.ts

```diff
@@ -71,12 +81,6 @@ export const contextParams: ContextParams = {
   ],
 };
 
-// Bare minimoum context that will use de default values
-const minimalContextParams: ContextParams = {
-  network: "mainnet",
-  web3Providers: "https://eth.llamarpc.com",
-  signer: Wallet.createRandom(),
-};
```

Comment:
Why removing this? That's what we recently added so that developers didn't need to enter full redundant settings

Reply:
Didn't remove it, moved it up top!